raiden/constants.py

```diff
@@ -165,10 +165,10 @@ class RoutingMode(Enum):
 
 
 # Keep in sync with .circleci/config.yaml
-HIGHEST_SUPPORTED_GETH_VERSION = "1.9.2"
+HIGHEST_SUPPORTED_GETH_VERSION = "1.9.8"
```

Comment:
This needs to be fixed

Reply:
To 1.9.9, yes. But didn't you merge a fix that ignores minor versions?

Reply:
Yes, it should run with higher versions anyways.